_posts/2015-04-30-gsoc-15-is-now-live.markdown

```diff
@@ -0,0 +1,123 @@
+---
+layout: post
+title: Google Summer of Code 2015 is now live
+categories:
+- activism
+author: febuiles
+published: true
+date: 2015-04-30 15:00:00 +00:00
+---
+**TODO** Confirm Josh K. participation and remove **TODO** tags.
+
+Google has announced the list of [accepted projects][melange] for this year's [Google Summer of
+Code][gsoc] (GSoC). Rails has been granted 8 slots, here's a brief introduction to the projects and
+the people behind them.
+
+## Performance Enhancements for the Asset Pipeline
+
+**Student:** [Martha de Luque]()    
+**Mentors:** [Guillermo Iguaran](https://github.com/guilleiguaran) and [Joshua Peek](https://github.com/josh)
+
+Martha will be profiling, benchmarking and updating parts of our asset compilation process to
+improve our asset [re]generation speed. The initial scope of this project covers CoffeeScript, Sass and
+our Uglifier, but benchmarks will be guiding this effort to work where we can get the biggest
+benefits in these four months.
+
+## Evented File System Monitoring
+
+**Student:** [Puneet Agarwal](https://github.com/puneet24)    
+**Mentors:** [Xavier Noria](https://github.com/puneet24) and [Matthew Draper](https://github.com/matthewd)
+
+[FileUpdateChecker](http://api.rubyonrails.org/classes/ActiveSupport/FileUpdateChecker.html), the
+system we use for detecting changes in files (mostly for reloading purposes) has served us well over the years, but we're done with
+polling. Puneet will be replacing our current design with a event-based approach that relies
+on existing third-party monitors (e.g. `inotify` or `FSEvent`).
+
+## Asset Pipeline Support for Source Maps
+
+**Student:** [Andrei Istratii](https://github.com/Andreis13)    
+**Mentors:** [Arthur Nogueira Neves](https://github.com/arthurnn) and [Rafael Mendonça França](https://github.com/rafaelfranca)
+
+The goal of Andrei's project is to give you good inspecting and debugging capabilities in environments
+where your code goes through various transformations (e.g. your CoffeeScript file being compiled to
+Javascript and then minified in your staging environment). With source maps you can use the existing
+tools your browser provides to do things like reading the CoffeeScript source or setting breakpoints
+on it.
+
+## Refactoring ActiveSupport and ActionView
+
+**Student:** [Islam Wazery](https://github.com/wazery)    
+**Mentors:** [Kir Shatrov](https://github.com/kirs) and [Carlos Antonio da Silva](https://github.com/carlosantoniodasilva)
+
+Islam is adding some of the things we should already have in Rails, liked named arguments
+ for ActionView helpers (goodbye counting commas!), or removing the ActiveSupport dependency on
```

Comment:
When used inside a normal English sentence, these Rails components/libraries are spelled with a space between the words, so "Action View" and "Active Support". (Few other occurrences of the same error below) :smile: